docs/cloud/managing-airbyte-cloud/review-connection-status.md

```diff
@@ -4,49 +4,75 @@ products: all
+:::
+
+To view the connection status:
 
 1. In the Airbyte UI, click **Connections**.
 
-2. Click a connection in the list to view its status.
+2. Each connection is listed alongside it's status. You can also filter for a specific status to see specific connections.
 
-| Status          | Description                                                                                                                                                                                                  |
-| --------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| On time         | The connection is operating within the expected timeframe expectations set by the replication frequency                                                                                                      |
-| On track        | The connection is slightly delayed but is expected to catch up before the next sync.                                                                                                                         |
-| Delayed         | The connection has not loaded data within the scheduled replication frequency. For example, if the replication frequency is 1 hour, the connection has not loaded data for more than 1 hour                  |
-| Error           | The connection has not loaded data in more than two times the scheduled replication frequency. For example, if the replication frequency is 1 hour, the connection has not loaded data for more than 2 hours |
-| Action Required | A breaking change related to the source or destination requires attention to resolve                                                                                                                         |
-| In Progress     | The connection is currently extracting or loading data                                                                                                                                                       |
-| Disabled        | The connection has been disabled and is not scheduled to run                                                                                                                                                 |
-| Pending         | The connection has not been run yet, so no status exists                                                                                                                                                     |
+| Status      | Description                                                                                         |
+| ----------- | --------------------------------------------------------------------------------------------------- |
+| **Healthy** | The most recent sync for this connection succeeded                                                  |
```

Comment:
From what I can tell, you can see the per-connection status on the main Connections page and then the per-stream status on the page for a single connection. The description for **Healthy** here mentions connections and the description for **Failed** mentions streams, but depending on which view you're looking at either of these statuses could apply to Connections or Streams, is that right?

Reply:
We actually only have a "Healthy" and "Failed" status for Connections, but not Streams. We do more derivation of a status for streams and the closest thing to "Healthy" we have is "On Time" for streams.  When a stream fails, it will get categorized into "On Track" , "Late" or "Error" depending on when the user sees it (note: we're planning on moving away from this as it's too complicated logic to understand, but that's what we have today). 

Hope that makes sense! I can explain more if that doesn't make sense. You called out correctly that I had a bad "Failed" description here that referred to streams when it should have said connection - just fixed that.